ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -161,18 +161,35 @@ export default class StatusDropdown extends React.Component {
   };
 
   copySummary = async () => {
-    const { filteredAlerts, alertSummary, frameworks } = this.props;
+    const { alertSummary, repoModel, filteredAlerts, frameworks } = this.props;
     const { browsertimeAlertsExtraData } = this.state;
     const textualSummary = new TextualSummary(
       frameworks,
       filteredAlerts,
       alertSummary,
-      true,
+      null,
       await browsertimeAlertsExtraData.enrichAndRetrieveAlerts(),
     );
+
+    const templateArgs = {
+      bugType: 'defect',
+      framework: getFrameworkName(frameworks, alertSummary.framework),
+      revision: alertSummary.revision,
+      revisionHref: repoModel.getPushLogHref(alertSummary.revision),
+      alertHref: `${window.location.origin}/perfherder/alerts?id=${alertSummary.id}`,
+      alertSummary: textualSummary.markdown,
+      alertSummaryId: alertSummary.id,
+    };
+
+    const templateText =
```

Comment:
This text needs to be present in the copied summary only if we have regressions.